ParisBNF/abb/BNFabb87.xml

```diff
@@ -0,0 +1,215 @@
+                                    <height unit="mm">375</height>
+                                    <width unit="mm">320</width>
+                                </extent>
+                            </supportDesc>
+                            <layoutDesc>
+                                <layout columns="3" writtenLines="46">   
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+                        <handDesc>
+                            <handNote xml:id="h1" script="Ethiopic">
+                                <seg type="script">19th-century script</seg> 
+                                <desc>A. d'Abbadie desribes the handwriting as <foreign xml:lang="fr">'fine et jolie'</foreign>.    
+                                </desc>
+                            </handNote>
+                        </handDesc>
+                        <additions>
+                            <list>
+                                <item xml:id="a1">
+                                    <locus target="#209v" ></locus>
+                                    <desc type="GuestText">According to Conti Rossini 
+                                        , there is a <foreign xml:lang="fr">'note en amharique, contenant des indications testimoniales sur un achat de terre non spécifié'</foreign>
+                                        <bibl>
+                                            <ptr target="bm:ContiRossini1914Notice"/>
+                                            <citedRange unit="page">199-120</citedRange>
+                                        </bibl> 
+                                    </desc>
+                                </item>
+                            </list>
+                        </additions>
+                        <bindingDesc cert="unknown">
+                            <binding contemporary="Ethiopian">                  
+                                <decoNote xml:id="b1" type="Other">A. d'Abbadie desribes the binding as <foreign xml:lang="fr">'belle reliure peu usée'.</foreign>. </decoNote>                  
+                                <decoNote xml:id="b7" type="SlipCase">According to A. d’Abbadie,
+                                    the manuscript has a simple two-part leather case.</decoNote>
+                      
+                            </binding>
+                        </bindingDesc>
+               
+                    </physDesc>
+                  
+                    <history>
+                            <origin>
+                                <origDate notBefore="1837" notAfter="1848" source="lettering">19th century</origDate>.  This manuscript was collected by  <persName
+                                    ref="PRS1071dAbbadi"/>A.d'Abbadie in Ethiopia.  </origin>
+                            <acquisition>The manuscript entered <placeName ref="INS0303BNF"/> in
+                                <date>1902</date>.</acquisition>
```

Comment:
If I understood correctly, it's better to give dates as `<date when="1902">1902</date>` for searchability reasons